apps/docs/src/app/core/component-docs/dialog/dialog-docs.component.html

```diff
@@ -0,0 +1,194 @@
+    <code>DialogRef</code> dependency inside the component to access Dialog methods and data. Data available in <code>DialogRef</code>
+    can be provided in <code>DialogConfig</code> object passed as an argument to <code>DialogService.open()</code>
+    method.
+</description>
+<component-example>
+    <fd-component-based-dialog-example></fd-component-based-dialog-example>
+</component-example>
+<code-example [exampleFiles]="componentAsContentSource"></code-example>
+
+<separator></separator>
+
+<fd-docs-section-title [id]="'dialogState'" [componentName]="'dialog'">
+    Dialog state
+</fd-docs-section-title>
+<description>
+    Dialog state can be controlled with <code>DialogRef</code>, providing such features like:
+    <ul>
+        <li><code>DialogRef.hide()</code> - visually hiding the Dialog window</li>
+        <li><code>DialogRef.close()</code> - closing the Dialog window</li>
+        <li><code>DialogRef.dismiss()</code> - dismissing the Dialog window</li>
+        <li><code>DialogRef.loading()</code> - displaying the dialog in loading state</li>
+    </ul>
+</description>
+<component-example>
+    <fd-dialog-state-example></fd-dialog-state-example>
+</component-example>
+<code-example [exampleFiles]="stateDialog"></code-example>
+
+<separator></separator>
+
+<fd-docs-section-title [id]="'dialogConfiguration'" [componentName]="'dialog'">
+    Dialog configuration
+</fd-docs-section-title>
+<description>
+    Dialog behavior can be configured using <code>DialogConfig</code> object passed as an argument to <code>DialogService.open()</code>
+    method. Passing <code>DialogConfig</code> overrides specified behaviours of the Dialog component.
+</description>
+<component-example>
+    <fd-dialog-configuration-example></fd-dialog-configuration-example>
+</component-example>
+<code-example [exampleFiles]="customConfiguration"></code-example>
+
+<description>
+    Default values used by the Dialog can be specified by providing <code>DIALOG_DEFAULT_CONFIG</code> token value.<br/><br/>
+    <code>
+        @NgModule({{'{'}}
+            provide: [
+                {{'{'}}provide: DIALOG_DEFAULT_CONFIG, useValue: customConfig{{'}'}}
+            ]
+        {{'}'}}
+    </code>
+</description>
+
+<separator></separator>
+
+<fd-docs-section-title [id]="'dialogPosition'" [componentName]="'dialog'">
+    Dialog positioning
+</fd-docs-section-title>
+<description>
+    The placement of the Dialog on the screen can be customized through the <code>DialogConfig.position</code> property.
+</description>
+<component-example>
+    <fd-dialog-position-example></fd-dialog-position-example>
+</component-example>
+<code-example [exampleFiles]="dialogPosition"></code-example>
+
+<separator></separator>
+
+<fd-docs-section-title [id]="'dialogMobile'" [componentName]="'dialog'">
+    Dialog on mobile devices
+</fd-docs-section-title>
+<description>
+    To display Dialog component in mobile mode set use <code>DialogConfig.mobile</code> property.
```

Comment:
To display the Dialog component in mobile mode, use the <code>DialogConfig.mobile</code> property.